src/ansys/additive/core/additive.py

```diff
@@ -352,6 +349,15 @@ def tune_material(
     ) -> MaterialTuningSummary:
         """Tune a custom material for use with additive simulations.
 
+        This method performs the same function as the Material Tuning Tool
```

Comment:
I assume "Material Tuning Tool" is initial caps in the documentation? Today, technical documentation avoids unnecessary capitalization, reserving it for proper nouns as it should! However, legacy Ansys product documentation uses initial capital letters everywhere--so I get it if you are matching the name (capitalization) in the existing documentation.

Reply:
Yes. The currently released docs (23.2) use initial caps.